ui/tank_warlock/presets.ts

```diff
@@ -229,7 +229,7 @@ export const DefaultConsumes = Consumes.create({
 	mainHandImbue: WeaponImbue.ShadowOil,
 	manaRegenElixir: ManaRegenElixir.MagebloodPotion,
 
-	sapper: true,
+	//sapper: true, Needs updated to new picker
```

Comment:
For these you should be able to set it to something like
```
sapper: Sapper.GoblinSapper
```
for now

Reply:
updated